gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2-entrypoints/step-2-entrypoints-0-architecture.component.ts

```diff
@@ -181,4 +181,39 @@ export class Step2Entrypoints0ArchitectureComponent implements OnInit, OnDestroy
       )
       .subscribe();
   }
+
+  onMoreInfoMessageClick(event) {
+    event.stopPropagation();
+
+    this.matDialog
+      .open<GioConnectorDialogComponent, GioConnectorDialogData, boolean>(GioConnectorDialogComponent, {
+        data: {
+          name: 'Message',
+          pluginMoreInformation: {
+            description: `
+              <p>
+                Includes asynchronous and event-driven API entrypoints. Typically used for streaming APIs. If you want to learn more about
+                Gravitee’s new capabilities, you can read about our Service Management Ecosystem.
```

Comment:
For the future, but this should link to where the user can read about the Service Management Ecosystem.

Reply:
Yes I hope we will have time to improve this later with feedback or when the doc will have a robust link on this part